tools/clang/test/lit.cfg

```diff
@@ -269,6 +269,41 @@ config.substitutions.append( ('%itanium_abi_triple', makeItaniumABITriple(config
 config.substitutions.append( ('%ms_abi_triple', makeMSABITriple(config.target_triple)) )
 config.substitutions.append( ('%dxc', lit.util.which('dxc', llvm_tools_dir)) )
 
+bin_dir = os.path.join(config.llvm_obj_root, lit_config.params['build_mode'], 'bin')
+config.environment['PATH'] = str.format('{};{}',config.environment['PATH'], bin_dir)
+
+
+# Displaying the parent directory of the script
+cur_dir = os.path.dirname(__file__)
+
+config.substitutions.append( ('%batch',
+                            'dxc_batch' ))
+
+config.substitutions.append( ('%dxv',
+                            'dxv' ))
+
+config.substitutions.append( ('%dxa',
+                            'dxa') )
+
+if platform.system() not in ['Windows']:
+    config.substitutions.append( ('%echo',
+                                    os.path.join(cur_dir, "echo.sh") ))
+else:
+    config.substitutions.append( ('%echo',
+                                    os.path.join(cur_dir, "echo.bat") ))
+
+config.substitutions.append( ('%opt',
```

Comment:
This substitution concerns me. We probably should use `dxopt` rather than `opt` to ensure we don't have confusion between `dxopt` and llvm's `opt`